install/operator/pkg/generator/assets/install/role.yml.tmpl

```diff
@@ -223,7 +224,7 @@
     - apps.openshift.io
     resources:
     - deploymentconfigs
-    - deploymentconfigs/scale
+    - deploymentconfigs/finalizers
```

Comment:
Don't we need this? If an addon is enabled / disabled, I think the operator has to scale up/down the deploymentconfig. Once the operator takes full control of the upgrade process, this will needed.